config/.env.example

```diff
@@ -1,7 +0,0 @@
-PORT = 2121 (Do not change)
-DB_STRING = your database URI
-CLOUD_NAME = your Cloudinary cloud name
-API_KEY = your Cloudinary API key
-API_SECRET = your Cloudinary API secret
-DISCORD_CLIENT_ID = discord client ID
-DISCORD_CLIENT_SECRET = discord client secret
```

Comment:
@Caleb-Cohen I noticed that a lot of PR has `.env.example` removed. I think we should add to the instructions; instead of renaming the `.env.example` to `.env` we should have people create a seperate `.env` file so that `.env.example` would not be deleted.

Reply:
Ah, that's why the .env has been getting deleted. Will do.